packages/dev/core/src/FlowGraph/flowGraphCoordinator.ts

```diff
@@ -15,9 +16,22 @@ export class IFlowGraphCoordinatorConfiguration {
     scene: Scene;
 }
 
+/**
+ * @experimental
+ * Parameters used to parse a flow graph coordinator.
+ */
 export class IFlowGraphCoordinatorParseOptions {

```

Comment:
Sorry, just randomly noticed this. Is this supposed to be an interface? Either interface or it should not have a `I` prefix

Reply:
Yup sorry it was indeed supposed to be an interface.